src/styles/blocks/header.scss

```diff
@@ -0,0 +1,59 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo-link {
+  display: flex;
+  cursor: pointer;
+
+  @include transition;
+}
+
+.nav__links {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+
+  .nav__link-item {
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  .nav__link {
+    position: relative;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+    font-style: normal;
+    color: #000;
+
+    @include transition;
+
+    &.nav__link--is-active,
+    &:hover {
+      color: var(--accent-color);
```

Comment:
Ensure that the `--accent-color` custom property is defined in your `variables.scss` file, otherwise this will not work as expected.